app/views/ForecastTaxCalcSummary.scala.html

```diff
@@ -34,6 +36,27 @@
     }
 }
 
+@dataItem2(message: String, value: Option[BigDecimal], value2: Option[BigDecimal], bold: Boolean = false) = {
+    @if(value2.isDefined && value.isDefined){
```

Comment:
* looks like a good case to apply pattern matching on 'value' and 'value2', 
* might be good idea to rename value2 to something more easy to understand/read, as you have assume code would be maintained in the future etc;